DataManagementSystem/DB/FileCatalogComponents/MultiVOMetaNameMixIn.py

```diff
@@ -0,0 +1,51 @@
+""" DIRAC Multi VO MixIn class to manage file metadata and directory for multiple VO.
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.DataManagementSystem.DB.FileCatalogComponents.MetaNameMixIn import MetaNameMixIn
+from DIRAC.ConfigurationSystem.Client.Helpers import Registry
+
+
+class MultiVOMetaNameMixIn(MetaNameMixIn):
+  """
+  MULti-VO MetaName MixIn implementation.
+  """
+
+  def __init__(self, *args, **kwargs):
+    super(MultiVOMetaNameMixIn, self).__init__(*args, **kwargs)
+
+  def getMetaName(self, meta, credDict):
+    """
+    Return a fully-qualified metadata name based on client-suplied metadata name and
+    client credentials. User group is added to the metadata passed in.
```

Comment:
It is the users VO that is added to the metadata name

Reply:
Yes.